clients/core/src/main/scala/mqperf/Metrics.scala

```diff
@@ -7,7 +7,7 @@ object Metrics {
 
   object Sender {
     val messageCounter: Counter =
-      Counter.build("mqperf_sent_total", "number of sent messages").labelNames(testIdLabelName).register()
+      Counter.build("mqperf_sent", "number of sent messages").labelNames(testIdLabelName).register()
```

Comment:
This represent all the sent messages during one test session? one execution? Wouldn't be a good idea to extend the description about this metric? And same above with `mqperf_received`?